anitya/lib/model.py

```diff
@@ -451,7 +451,15 @@ def updated(
             query = query.filter(
                 Project.logs != None,
                 Project.logs != 'Version retrieved correctly',
+                ~Project.logs.ilike('Something strange occured%'),
```

Comment:
It's the negation, so in this case I do not want the project whose log start with 'Something strange occured'